scripts/Interface/ui/drill/model/DrillModel.py

```diff
@@ -444,7 +444,7 @@ def changeParameter(self, sampleIndex, name, value):
             value (str): new value
         """
         self.samples[sampleIndex].changeParameter(name, value)
-        if (value == "DEFAULT") or not value:
+        if (value == "DEFAULT") or value == "":
```

Comment:
That's the minor comment, but I believe this could be replaced with a test of membership: `if value in ["DEFAULT", ""]`.